### PR TITLE
fix: indent in `heredoc`

### LINKED ISSUE #209: Auto-indenting heredoc
Input:

```php
<?php
class test {
    public function get_string(){
        $string = <<<EOJS
some text
should not indent
EOJS;
        return $string;
    }
}
```
Output:
```php
<?php
class test
{
    public function get_string()
    {
        $string = <<<EOJS
        some text
should not indent
EOJS;
        return $string;
    }
}
```
Expected output:
```php
<?php
class test {
    public function get_string(){
        $string = <<<EOJS
some text
should not indent
EOJS;
        return $string;
    }
}
```

### PR DESCRIPTION
**Pull request by evilebottnawi**
fixes #209

### CODE REVIEW

**Comment by evilebottnawi** [comment]
Also need improve tests

File: tests/encapsed/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -11,6 +11,28 @@ $encapsHereDoc2 = <<<STR
 Hello foo $a bar
 new line
 STR;
+echo <<<"FOOBAR"
```
Comment (evilebottnawi) [inline tests/encapsed/__snapshots__/jsfmt.spec.js.snap]: Issue: https://github.com/glayzzle/php-parser/issues/133

**Comment by evilebottnawi** [comment]
@mgrip @czosel ready for review :+1:

**Comment by mgrip** [comment]
can you add a test that has some indentation as well?

```php
<?php
class test {
    public function get_string(){
        $string = <<<EOJS
         some text
should not indent
EOJS;
        return $string;
    }
}
```

**Comment by evilebottnawi** [comment]
@mgrip done :+1:

**Comment by mgrip** [comment]
excellent 🎉

**Pull request merged**